CfgLoadouts.hpp

```diff
@@ -23,6 +23,10 @@ class CfgLoadouts {
   // #include "Loadouts\blu_swede_ak5_m90.hpp"     // Swedish: AK5 - M90 (Green)
   // #include "Loadouts\blu_ger_g36_fleck.hpp"     // German: G36 - Flecktarn Camo
 
+  //USMC FACTION (rhs_faction_usmc_wd):
+  #include "Loadouts\blu_usmc_m16_desert.hpp"      // USMC COOP Template Desert Loadout
```

Comment:
the file name should be:
usmc_us_m16_desert.hpp

the comment should be:
// US: M16 - FROG Desert